content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md

```diff
@@ -521,7 +521,214 @@ To verify your device status, click on your FoundriesFactory, go to **Devices**
+## Portenta X8 OS Image Update
+
+It is recommended to check every now and then if your Portenta X8 image version is up to date, in order to have the latest security updates.
+
+In the next sections, four major ways to update your Portenta X8 are described:
+
+* Update through Out-of-the-box experience (available for OS release XXXX or newer)
+* Update through Portenta X8 Manager in your Arduino Cloud for Business account (available for all OS releases)
+* Update for OS release V.399
+* Update using `uuu` tool (compatible with custom images)
+
+### Check Portenta X8 OS Release
+
+In order to verify which OS release is flashed on your Portenta X8, you need to connect to your board through **ADB**, as explained in [this section](#working-with-linux) of this user manual.
+
+At this point, you can type `cat /etc/os-release` in your command line window to get the OS release currently running on your device.
+
+![Get OS release](assets/adb-shell-os-release.png "Get OS Release")
+
+As shown in the image above, the OS release of this Portenta X8 corresponds to `IMAGE_VERSION=569`.
+
+### Update Through Out-Of-The-Box Experience
+
+Leverage the integrated Out-of-the-box experience to update your Portenta X8 to the latest release.
+
+***Warning: The Out-of-the-box update feature is not a complete Over-The-Air (OTA) update, it allows the user to update only Portenta X8 default image and containers. It will overwrite any custom container application. Thus, it is recommended to make a local copy of your containers before updating your Portenta X8.***
+
+Open your Out-of-the-box as explained in [this section](#first-use-of-your-portenta-x8).
+
+![Out-of-the-box homepage](assets/OOTB_homepage.png "Out-of-the-box homepage")
+
+Click on **CHECK FOR UPDATES** in the lower right corner.
+
+At this point, you have to select whether you would like to proceed with the update. If yes, click on **UPDATE**.
+
+![Proceed with update](assets/OOTB_update_select.png "Proceed with update")
+
+During the update, do not turn off your Portenta X8 or disconnect it from the network. This process may take few minutes.
+
+![Successful update](assets/OOTB-succesful-OS-update.png "Successful update")
+
+Once the update is finished, your Portenta X8 will automatically restart with the new Linux image in place.
+
+At this point, if you would to continue to use your Out-of-the-box, you can open a new command line window and launch again the command `adb forward tcp:8080 tcp:80`. Now open your browser, go to [http://localhost:8080](http://localhost:8080) and the same Out-of-the-box dashboard will appear.
+
+#### Troubleshooting
+
+If something gets wrong during the update, you still have the possibility to manually flash your Portenta X8 with the latest Linux image provided at [this link](https://github.com/arduino/lmp-manifest/releases). Follow [this tutorial](https://docs.arduino.cc/tutorials/portenta-x8/image-flashing) to learn how to flash your device manually.
```

Comment:
Here you can also link to the "Update using 'uuu' command" you added if you like

Reply:
Good point, I'll have it added.